.travis.yml

```diff
@@ -33,29 +33,23 @@ before_script:
   - chmod a+rwx ${HOST_COVERAGE_DATA_DIR}
 
 script:
-  - docker-compose -f docker-compose.yml -f docker-compose.ci.yml run cvat_ci /bin/bash -c 'coverage run -a manage.py test cvat/apps utils/cli && mv .coverage ${CONTAINER_COVERAGE_DATA_DIR}'
-  - docker-compose -f docker-compose.yml -f docker-compose.ci.yml run cvat_ci /bin/bash -c 'cd cvat-data && npm install && cd ../cvat-core && npm install && npm run test && coveralls-lcov -v -n ./reports/coverage/lcov.info > ${CONTAINER_COVERAGE_DATA_DIR}/coverage.json'
+  - docker-compose -f docker-compose.yml -f docker-compose.ci.yml run cvat_ci /bin/bash -c 'coverage run -a manage.py test cvat/apps utils/cli'
+  - docker-compose -f docker-compose.yml -f docker-compose.ci.yml run cvat_ci /bin/bash -c 'cd cvat-data && npm install && cd ../cvat-core && npm install && npm run test'
   - docker-compose up -d
   # Create superuser
   - docker exec -it cvat bash -ic "echo \"from django.contrib.auth.models import User; User.objects.create_superuser('${DJANGO_SU_NAME}', '${DJANGO_SU_EMAIL}', '${DJANGO_SU_PASSWORD}')\" | python3 ~/manage.py shell"
   # End-to-end testing
   - if [[ $TRAVIS_EVENT_TYPE == "cron" && $TRAVIS_BRANCH == "develop" ]];
     then
+      npm install &&
+      npm run coverage &&
+      docker-compose up -d --build &&
       cd ./tests && npm install &&
-      npm run cypress:run:firefox && cd ..  &&
-      npm install  &&
-      npm run coverage  &&
-      docker-compose up -d --build  &&
-      cd ./tests &&
-      npm run cypress:run:chrome && cd .. &&
+      npm run cypress:run:firefox && cd .. &&
+      npm run move:nyc_output &&
       npm run report:coverage:text &&
       npm run report:coverage:lcov;
     else
       cd ./tests && npm install &&
       npm run cypress:run:chrome && cd ..;
     fi;
-
-
-after_success:
-  # https://coveralls-python.readthedocs.io/en/latest/usage/multilang.html
-  - docker-compose -f docker-compose.yml -f docker-compose.ci.yml run cvat_ci /bin/bash -c 'ln -s ${CONTAINER_COVERAGE_DATA_DIR}/.git . && ln -s ${CONTAINER_COVERAGE_DATA_DIR}/.coverage . && ln -s ${CONTAINER_COVERAGE_DATA_DIR}/coverage.json . && coveralls --merge=coverage.json'
```

Comment:
I've just realized that the coverage is computed also for a server part. Could you please investigate if we could have two different badges for client and for server? If it is impossible, anyway we need left coverage for server part as is and merge results for the same badge (maybe also we should to run server coverage only on cron builds)

Reply:
Well. I will investigate this topic. It will probably be easier to combine the results.